### PR TITLE
Setting up WPCS in VS Code

### PR DESCRIPTION
**Pull request by kuldeep3**
**Changes done**

1. Added `readme` for how to setup `WPCS` on visual studio code for `windows`.

### CODE REVIEW

File: wordpress/WPCS.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Requirements
+
+1. Visual Studio Code
+1. PHP
+1. Composer
+
+### Installing Visual Studio Code
+
+1. [Download Visual Studio Code](https://code.visualstudio.com/Download)
+1. Run the installer.
```
Comment (abhishek-pokhriyal) [inline wordpress/WPCS.md:16]: Since Visual Studio Code is a prerequisite, we can get rid of this part.

**Review by abhishek-pokhriyal — COMMENTED** [review]
@kuldeep3 Left a few suggestions.

**Comment by Satendra-SR** [comment]
@kuldeep3 any progress on this one?

**Comment by kuldeep3** [comment]
> 
> 
> @kuldeep3 any progress on this one?

@Satendra-SR I am targetting it to get complete by this weekend

**Review by abhishek-pokhriyal — APPROVED** [review]
@kuldeep3 Looking good. Can you also add an entry in the main README file in the Wordpress folder so that it is accessible via coloredcow.github.io/resource?

**Pull request merged**